### PR TITLE
🌐 N°7247 - Update italian translations

### PR DESCRIPTION
**Pull request by DarkNight97boss**

### CODE REVIEW

**Review by Molkobain — CHANGES_REQUESTED** [review]
Thanks for the PR and the huge amount of translations @DarkNight97boss !

Just 2 things to correct please:
- Revert changes to `datamodels/2.x/authent-cas/src/CASLogger.php` as it is not part of the translations and was already done in #593 
- For each dict entry translated in italian, please remove the trailing `~~`. These symbols are only there for un-translated entries to help translators find new entries.

File: datamodels/2.x/authent-cas/dictionaries/it.dict.authent-cas.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * @license     http://opensource.org/licenses/AGPL-3.0
  */
 Dict::Add('IT IT', 'Italian', 'Italiano', array(
-	'CAS:Error:UserNotAllowed' => 'User not allowed~~',
-	'CAS:Login:SignIn' => 'Sign in with CAS~~',
-	'CAS:Login:SignInTooltip' => 'Click here to authenticate yourself with the CAS server~~',
+	'CAS:Error:UserNotAllowed' => 'Utente non autorizzato~~',
+	'CAS:Login:SignIn' => 'Accedi con CAS~~',
+	'CAS:Login:SignInTooltip' => 'Clicca qui per autenticarti con il server CAS~~',
```
Comment (Molkobain) [inline datamodels/2.x/authent-cas/dictionaries/it.dict.authent-cas.php]:
```suggestion
	'CAS:Error:UserNotAllowed' => 'Utente non autorizzato',
	'CAS:Login:SignIn' => 'Accedi con CAS',
	'CAS:Login:SignInTooltip' => 'Clicca qui per autenticarti con il server CAS',
```

Example of entries that should no longer have trailing `~~`

**Comment by DarkNight97boss** [comment]
> Thanks for the PR and the huge amount of translations @DarkNight97boss !
> 
> Just 2 things to correct please:
> 
> * Revert changes to `datamodels/2.x/authent-cas/src/CASLogger.php` as it is not part of the translations and was already done in [🐛  Fixing error during setup #593](https://github.com/Combodo/iTop/pull/593)
> * For each dict entry translated in italian, please remove the trailing `~~`. These symbols are only there for un-translated entries to help translators find new entries.

Done!

can you remove unnecessary trailing yourself? if not, I can do it for you! but i need some days

**Comment by Molkobain** [comment]
It would be best if you could do it yourself, thanks. No hurry though, don't worry :)

File: datamodels/2.x/authent-ldap/dictionaries/it.dict.authent-ldap.php [hunk 2]
```diff
@@ -36,5 +36,5 @@
 Dict::Add('IT IT', 'Italian', 'Italiano', array(
 	'Class:UserLDAP' => 'Utente LDAP',
 	'Class:UserLDAP+' => 'Utente autenticato da LDAP',
-	'UserLDAP:server' => 'LDAP specifics~~',
+	'UserLDAP:server' => 'Specifiche LDAP~~',
```
Comment (Molkobain) [inline datamodels/2.x/authent-ldap/dictionaries/it.dict.authent-ldap.php]:
```suggestion
	'UserLDAP:server' => 'Specifiche LDAP',
```

File: datamodels/2.x/authent-local/dictionaries/it.dict.authent-local.php [hunk 3]
```diff
@@ -37,20 +37,17 @@
...
+	'Class:UserLocal/Attribute:expiration/Value:can_expire' => 'Può scadere~~',
+	'Class:UserLocal/Attribute:expiration/Value:never_expire' => 'Non scade~~',
+	'Class:UserLocal/Attribute:expiration/Value:force_expire' => 'Scaduta~~',
+	'Class:UserLocal/Attribute:expiration/Value:otp_expire' => 'Password monouso~~',
+	'Class:UserLocal/Attribute:expiration/Value:otp_expire+' => 'La password non può essere cambiata dall\'utente.~~',
+	'Class:UserLocal/Attribute:password_renewed_date' => 'Rinnovo della password~~',
+	'Class:UserLocal/Attribute:password_renewed_date+' => 'Quando è stata cambiata l\'ultima volta la password~~',
+	'Error:UserLocalPasswordValidator:UserPasswordPolicyRegex:ValidationFailed' => 'La password deve essere di almeno 8 caratteri e includere lettere maiuscole, minuscole, numeri e caratteri speciali.~~',
+	'UserLocal:password:expiration' => 'I campi sottostanti richiedono un\'estensione~~',
+	'Class:UserLocal/Error:OneTimePasswordChangeIsNotAllowed' => 'Impostare la scadenza della password su "Password monouso" non è consentito per il proprio utente~~',
```
Comment (Molkobain) [inline datamodels/2.x/authent-local/dictionaries/it.dict.authent-local.php]:
```suggestion
	'Class:UserLocal/Attribute:expiration' => 'Scadenza della password',
	'Class:UserLocal/Attribute:expiration+' => 'Stato della scadenza della password (richiede un\'estensione per avere effetto)',
	'Class:UserLocal/Attribute:expiration/Value:can_expire' => 'Può scadere',
	'Class:UserLocal/Attribute:expiration/Value:never_expire' => 'Non scade',
	'Class:UserLocal/Attribute:expiration/Value:force_expire' => 'Scaduta',
	'Class:UserLocal/Attribute:expiration/Value:otp_expire' => 'Password monouso',
	'Class:UserLocal/Attribute:expiration/Value:otp_expire+' => 'La password non può essere cambiata dall\'utente.',
	'Class:UserLocal/Attribute:password_renewed_date' => 'Rinnovo della password',
	'Class:UserLocal/Attribute:password_renewed_date+' => 'Quando è stata cambiata l\'ultima volta la password',
	'Error:UserLocalPasswordValidator:UserPasswordPolicyRegex:ValidationFailed' => 'La password deve essere di almeno 8 caratteri e includere lettere maiuscole, minuscole, numeri e caratteri speciali.',
	'UserLocal:password:expiration' => 'I campi sottostanti richiedono un\'estensione',
	'Class:UserLocal/Error:OneTimePasswordChangeIsNotAllowed' => 'Impostare la scadenza della password su "Password monouso" non è consentito per il proprio utente',
```

**Review by Molkobain — CHANGES_REQUESTED** [review]

**Comment by Molkobain** [comment]
Technical review:
  * Dict entries must be clean to remove trailing `~~`
  * Guillaume will handle the cleaning and rebase the branch on support/3.2
  * Then we can move the PR to functional review

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Why did you merge develop in this branch @DarkNight97boss ? It did a big mess.
Don't touch the PR branch, I'll fix it. BTW we fixed the remaining issues yesterday.

**Review by Molkobain — CHANGES_REQUESTED** [review]
Need to revert last merge from develop branch

**Comment by Molkobain** [comment]
Accepted during functional review. I'll fix the branch and merge in the coming days.

**Comment by DarkNight97boss** [comment]
> Accepted during functional review. I'll fix the branch and merge in the coming days.

Great! sounds good... about data sync issue?

**Comment by Molkobain** [comment]
I'll handle it, don't worry :)

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
@DarkNight97boss can you send me your full name and postal address to guillaume.lajarige @ combodo.com please? We would like to send you [thankful stickers](https://github.com/Combodo/iTop/blob/develop/.doc/contributing-guide/2023.contributing-stickers-side-by-side.png) for your contributions :)

**Pull request merged**